widgetssdk/src/main/java/com/glia/widgets/call/CallView.java

```diff
@@ -773,7 +773,9 @@ private void handleStatusbarColor() {
     }
 
     private void showEndEngagementDialog(String operatorName) {
-        showOptionsDialog(
+        alertDialog = Dialogs.showOptionsDialog(
```

Comment:
Do I understand correctly that `showOptionsDialog` function is not in use anymore?

Reply:
"Unexpected error" and "no more operators available" dialogs are using those